extensions/ql-vscode/src/common/github-url-identifier-helper.ts

```diff
@@ -0,0 +1,71 @@
+import { OWNER_REGEX, REPO_REGEX } from "../pure/helpers-pure";
+
+/**
+ * Checks if a string is a valid GitHub NWO.
+ * @param identifier The GitHub NWO
+ * @returns
+ */
+export function validGitHubNwo(identifier: string): boolean {
```

Comment:
Minor (feel free to ignore if this would make conflicts etc annoying!) 👍🏽 
I'd probably name this function "isValid..." since that sounds more like a true/false check. Similar for the other functions :) 

```suggestion
export function isValidGitHubNwo(identifier: string): boolean {
```